DataCollection/src/Infrastructure/Persistence/DataCollectionContext.cs

```diff
@@ -54,6 +55,24 @@ public class DataCollectionContext : DbContext, ILocationPersistence, IWorkLogPe
         }
         #endregion
 
+        #region Specialized Queries for IBirdSpeciesPersistence
+
+        public IEnumerable<BirdSpecies> GetBirdSpeciesBySurveyTypeId(int surveyTypeId)
+        {
+            var returnVal = new List<BirdSpecies>();
+            var surveyType = SurveyTypes.Include(survey => survey.BirdSpecies).FirstOrDefault(surveyItem => surveyItem.Id == surveyTypeId);
+
+            if (surveyType != null)
+            {
+                if (surveyType.BirdSpecies != null)
+                {
+                    returnVal = surveyType.BirdSpecies.ToList();
+                }
+            }
+            return returnVal;
```

Comment:
I'm going to move forward with the pull request anyway, but please do go ahead and move the code.